### ISSUE TITLE
NVDA doesn't announce that multiple checkbox was checked in Chrome

### ISSUE THREAD

**Issue by VasylRudnitskyi** [nvaccess/nvda#9322]
### Steps to reproduce:
1. Create html page with some amount of checkboxes (in my case reproduced after 12)
2. Make JavaScript function that will check all of them after some of them will be clicked
3. Open page via Chrome, start NVDA and check this checkbox

### Actual behavior:
NVDA doesn't say that checkbox was checked
### Expected behavior:
NVDA should say that checkbox was checked

#### NVDA installed/portable/running from source:
running from source
#### NVDA version:
2018.3.2
#### Windows version:
Windows 10 Pro 1803
#### Name and version of other software in use when reproducing the issue:
Chrome version 72.0.3626.119

#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA?
No

This Bug is reproducing only for NVDA + Chrome in other combination works good.
I've opened similar issue for Chrome:
https://bugs.chromium.org/p/chromium/issues/detail?id=922020

Attached repro in txt format (need change to html)
[repro.txt](https://github.com/nvaccess/nvda/files/2900560/repro.txt)

**Comment by Adriani90** [comment]
This is also reproducible in Firefox 76 I think. NVDA reports randomly the correct state when checking the states of the checkboxes and when pressing space bar on some of them.
I am not sure if this is a web authoring issue here.
cc: @jcsteh

**Comment by jcsteh** [comment]
Duplicate of #10484. Even though this one is older, closing as a duplicate of the other issue because it has more technical detail and discussion.
